frontend/public/components/utils/dom-utils.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+
+const isHTMLElement = (n: Node): n is HTMLElement => {
+  return n.nodeType === Node.ELEMENT_NODE;
+};
+
+export const getParentScrollableElement = (node: HTMLElement) => {
+  let parentNode: Node = node;
+  while (parentNode) {
+    if (isHTMLElement(parentNode)) {
+      let overflow = parentNode.style.overflow;
+      if (overflow !== 'scroll' && overflow !== 'auto') {
+        overflow = window.getComputedStyle(parentNode).overflow;
+      }
+      if (overflow === 'scroll' || overflow === 'auto') {
+        return parentNode;
```

Comment:
Is there a concern that somehow (likely accidental) this element doesn't have a height? Did you want to getBoundingBox and verify it has a height?